sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/test/public/instrumenter.spec.ts

```diff
@@ -270,7 +270,7 @@ describe("OpenTelemetryInstrumenter", () => {
 
       // Function syntax
       instrumenter.withContext(context.active(), function (this: any) {
-        assert.notExists(this);
+        assert.isDefined(this);
```

Comment:
This seems weird to me, but I don't want to block the PR. Feel free to pend the test and assign an issue to me to review it more closely. It's been like 2 years 😄

Reply:
yeah feels like it should be undefined as it's not being explicitly bound to anything?